app/src/lang/translations/en-US.yaml

```diff
@@ -66,7 +66,7 @@ indeterminate: Indeterminate
 edit_collection: Edit Collection
 exclusive: Exclusive
 children: Children
-db_only_click_to_configure: 'Database Only: Click to Configure'
```

Comment:
I am having issues with trying to mimic my local vscode to match the settings for this repo. But without a `.vscode/settings.json` I have been trying to work around it. Not sure if there are suggestions on how to fix these linting/editor formatter issues to match what the Directus repo needs.

Reply:
Do you happen to have the `editorconfig` extension installed? That file should control the basic formatting settings. Everything else is controlled through the auto-linter 🤔